pkgpanda/build/__init__.py

```diff
@@ -111,7 +111,7 @@ def get_src_fetcher(src_info, cache_dir, working_directory):
 
         return pkgpanda.build.src_fetchers.all_fetchers[kind](**args)
     except ValidationError as ex:
-        raise BuildError("Validation error when fetching sources for package: {}".format(ex))
+        raise BuildError("Validation error when fetching sources for package: {}".format(src_name)) from ex
```

Comment:
We want the exception here (it tells us _why_ the fetch failed). Please don't make these sorts of changes when there's so much code motion already. Should be separate PRs.